app/assets/stylesheets/application.scss

```diff
@@ -61,3 +61,32 @@ p {
   background-color: $l-gold;
   margin: 20px;
 }
+
+.house-list ul li{
+  display: inline;
+  padding-right: 10px;
+}
+
+.bill-list ul li{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space